.eslintrc.js

```diff
@@ -174,6 +174,11 @@ module.exports = {
             "ignoreReadBeforeAssign": false
         }],
 
+        // These were deprecated out of eslint-all and moved to @stylistic plugin.
+        // will be changed once we will start using the plugin as well
+        'no-multi-spaces': ['error', {}],
+        'semi': ['error', 'always'],
```

Comment:
@jackyalbo I think there are more cases, for example:

> /Users/shiradymnik/SourceCode/noobaa-core/src/test/unit_tests/test_bucketspace_fs.js
>   158:20  error  Extra space after key '_id'  key-spacing

It is `key-spacing`